src/components/CourseTable/index.jsx

```diff
@@ -166,7 +159,7 @@ class CourseTable extends React.Component {
               }
             />
           </div>
-          <div className="col-5 float-right">
+          <div className="px-3 float-left" style={{ width: '44%' }}>
```

Comment:
Either add these styles in an `App.scss` as classes and include them here. Inline css is a bad practice with React

Reply:
done